inc/admin/class-network-usage-columns.php

```
@@ -0,0 +1,365 @@
+	 */
+	public function manage_themes_custom_column(string $column_name, string $theme_key, WP_Theme $theme_data): void {
+		if ( 'active_blogs' !== $column_name ) {
+			return;
+		}
+
+		$active_on_blogs = $this->get_blogs_with_theme($theme_key);
+
+		$this->output_blog_list($active_on_blogs);
+
+		// Check, if is a child theme and return parent.
+		$child_context = '';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$child_context'. (undefined)

(UnusedLocalVariable)